extension/docs/CancelPayment.md

```diff
@@ -0,0 +1,108 @@
+**Please see [Integration Guide](WebComponentsIntegrationGuide.md) first before continuing with this document.**
+
+## Cancel a payment
+If you have authorised a payment but do not want to capture it, you need to make a Cancel request to release the funds back to the shopper.
+If the payment has already been captured, you need to [refund it](./Refund.md) instead.
```

Comment:
```suggestion
If you have authorised payment but do not want to capture it, then you can make a cancel request to release the funds back to the shopper.
If the payment has already been captured, then you should [refund it](./Refund.md) instead.
```

Reply:
This is copied from Adyen 😊 I guess they have it professionally written, so I won't make your fixes. https://docs.adyen.com/checkout/cancel (second paragraph).